examples/interactivemarkers.html

```diff
@@ -23,7 +23,8 @@
       divID : 'markers',
       width : 800,
       height : 600,
-      antialias : true
+      antialias : true,
+      alpha:0.5
```

Comment:
Huh? Sorry, I'm a little confused. Are you talking about consistency of the framework? The parameter is optional. I just put it in an example to show that it is there. I can remove the line if you like.